sdk/lib/3rdparty/libsamplerate/samplerate.c

```diff
@@ -6,7 +6,15 @@
 ** file at : https://github.com/erikd/libsamplerate/blob/master/COPYING
 */
 
-#include "precomp.h"
+#include	<stdio.h>
+#include	<stdlib.h>
+#include	<string.h>
+
+#include	"config.h"
+
+#include	"samplerate.h"
+#include	"float_cast.h"
+#include	"common.h"
```

Comment:
Isn't it possible to use again the precompiled header? (if needed, to add in it the missing new headers)

Reply:
Yup, I was just keeping the files as original from source :)